docs/Splashkit/Applications/mingw-automated installation/readme.md

```diff
@@ -0,0 +1,39 @@
+# Installation Script Documentation
+
+## Overview
+This script automates the installation process for SplashKit, .NET, C++ tools, and Visual Studio Code (VS Code) on Windows systems using MSYS2. It ensures a seamless setup of development tools required for software development.
+
+## Script Details
+The script is divided into several sections, each responsible for a specific task:
+
+1. **Update package databases:** Ensures that package databases are up to date before proceeding with installations.
+2. **Install required packages:** Installs necessary packages such as git, curl, and unzip.
+3. **Install C++ tools:** Installs essential C++ development tools using pacman.
+4. **Install .NET SDK:** Downloads and installs the .NET SDK using PowerShell to handle the installation process.
+5. **Install VS Code:** Downloads and installs VS Code using PowerShell to handle the installation process.
+6. **Install SplashKit:** Downloads and installs SplashKit using the provided installation script.
+7. **Install VS Code extensions:** Installs essential VS Code extensions for C++ and .NET development.
+
+## Usage
+To use the script, follow these steps:
+
+1. Ensure that you have MSYS2 installed on your system.
+2. Copy the script to your system or download it. Save it as `install.sh` inside your MSYS2 home directory (`C:/msys/home/{username}/install.sh`).
+3. Open a MSYS2 bash terminal window.
```

Comment:
Which version? MSYS2 or MINGW64?

Reply:
- It has been updated to MINGW64 in readme.